docs/ReleaseNotes.md

```diff
@@ -26,6 +26,8 @@ Release Notes
           attempting to load the package just named just `x`
 - updated dependencies
 - fixed to set the formatter which designated from the command line option.
+- added an option `output` to write the output to a file.
```

Comment:
v2.0.1 is already published, so please make a new section for v2.1.0

Reply:
I update it.